app/controllers/case_contacts_controller.rb

```diff
@@ -83,7 +83,7 @@ def create
     elsif @selected_cases.count > 1
       redirect_to case_contacts_path(success: true)
     else
-      redirect_to casa_case_path(CaseContact.last.casa_case, success: true)
+      redirect_to casa_case_path(CaseContact.last.casa_case, success: true), notice: "Case contact created successfully."
```

Comment:
Is this correct? What is the rationale here?

Reply:
The spec was checking for a success message to be received. There was none in actual action so added this to make the test pass.